packages/block-library/src/buttons/edit.native.js

```diff
@@ -45,11 +46,13 @@ function ButtonsEdit( {
 		}
 	}, [ sizes ] );
 
+	const debounceAddNextButton = debounce( onAddNextButton, 500 );
```

Comment:
I have the impression that the value of `500` is a bit too big and wondering if `200` won't be enough? Could you please check that?

Reply:
I was able to reproduce this up to 170ms. 200 seems quite reasonable.
I updated the PR

Reply:
Thanks!